Docs/LinuxOsxInstall.md

```diff
@@ -24,9 +26,11 @@ Once the package is installed, it can be run with `factorio-yafc`. Note that at
   - `sudo apt-get install libsdl2-ttf-2.0-0`
   - For reference, have following libraries: SDL2-2.0.so.0, SDL2_ttf-2.0.so.0, SDL2_image-2.0.so.0
 - Make sure you have OpenGL available
-- To run either use `dotnet YAFC.dll` in the terminal, or run `YAFC` as an executable
+- Make `YAFC is executable with `chmod +x YAFC`
```

Comment:
```suggestion
- Make `YAFC` executable with `chmod +x YAFC`
```
Was that a typo, or am I missing something?

Reply:
typo :)